helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -721,6 +716,21 @@ void reset(boolean isShutdown) {
     }
   }
 
+  /**
+   * The package private method only closes the batchCallbackProcessor thread
+   * *Caution*: currently it's only used during disconnecting from ZK and the
+   * listeners unsubscription will be taken care by zkClient directly
+   */
+  void closeCallbackProcessor() {
```

Comment:
More precisely, it only closes batchProcessor. Rename the method

Reply:
done.